.circleci/config.yml

```diff
@@ -76,28 +23,35 @@ jobs:
             - run: mkdir -p test_preparation
             - run: python utils/tests_fetcher.py | tee tests_fetched_summary.txt
             - store_artifacts:
-                path: ~/transformers/tests_fetched_summary.txt
+                  path: ~/transformers/tests_fetched_summary.txt
```

Comment:
Do we deliberately not to store `examples_tests_fetched_summary` below? It was stored before this PR.

Reply:
There have been many changes in the config file since I did this PR (we were waiting for you to be back!) so I think I have some serious rebasing to do :-)